src/modules/kpi-reporting/services/data-service.js

```diff
@@ -0,0 +1,58 @@
+
+const crm = require('../../../lib/connectors/crm/kpi-reporting');
+const idm = require('../../../lib/connectors/idm/kpi-reporting');
+const events = require('../../../lib/services/events');
+const returns = require('../../../lib/connectors/returns');
+
+const errorHandler = (error) => {
+  if (error.statusCode === 404) {
+    return null;
+  }
+  return error;
+};
+
```

Comment:
Looks like codeclimate does not like some of the repetition here.

Could try something like

const getDataFromResponseEnvelope = async connectorFunc => {
  try {
    const { data } = await connectorFunc()
    return data
  } catch (err) {
    return err.statusCode === 404 : null ? err
  }
}

const getIDMRegistrationsData = async () => getDataFromResponseEnvelope(idm.getKpiRegistrations)
const getIDMRegistrationsData = async () => getDataFromResponseEnvelope(idm.getKpiAccessRequests)

```

Reply:
done